master/types.html

```diff
@@ -359,18 +359,8 @@ <h3 id="ElementsInTheSVGDOM">Elements in the SVG DOM</h3>
 must also implement the DOM interface identified in the element definition.
 </p>
 </li>
-<li>
-<p>
-Elements in the SVG namespace whose local name does not match an element
-defined in any specification supported by the software
-should instead implement the <a>SVGUnknownElement</a> interface and
-must nonetheless implement the <a>SVGElement</a> interface.
```

Comment:
The must part of this needs to be preserved! (It describes current browser behavior). Just delete line 366.

Reply:
Kept (with the one-line deletion as suggested by @AmeliaBR )